GuessWord/Startup.cs

```diff
@@ -46,6 +48,15 @@ public void ConfigureServices(IServiceCollection services)
                 options.MinimumSameSitePolicy = SameSiteMode.None;
             });
 
+            // Configure Data Protection to store keys in Google Cloud Storage, encrypted using Google Cloud KMS.
+            // These keys are then used to provide anti-forgery tokens which are consistent across servers.
+            var storageOptions = Configuration.GetSection("GoogleCloudStorageOptions").Get<GoogleCloudStorageOptions>();
+            var kmsOptions = Configuration.GetSection("GoogleCloudKmsOptions").Get<GoogleCloudKmsOptions>();
```

Comment:
This looks really pretty!

Reply:
This part isn't *too* bad - particularly if we put `GoogleCloudStorageOptions` as a class in `Google.Cloud.AspNetCore.DataProtection.Storage`, and add an overload that allows it to be passed in directly.
But I'm still concerned about the lack of being able to specify a `StorageClient` or credentials. I'm hoping to ask a Stack Overflow question today.